src/types/events/base-events.ts

```diff
@@ -227,6 +227,21 @@ export interface AppMentionEvent {
   bot_profile?: BotProfile;
   username?: string;
   team?: string;
+  // user_team, source_team, and user_profile
+  // can exist when the user who mentioned this bot is in a different workspace/org
+  user_team?: string;
+  source_team?: string;
+  user_profile?: {
+    name: string,
+    first_name: string,
+    real_name: string,
+    display_name: string,
+    team: string,
+    is_restricted?: boolean,
+    is_ultra_restricted?: boolean
```

Comment:
😳 woah this is intense! Also wondering if it should match the commas or semicolons for the object but not a blocker at all. This is something the linter should probably know 💭

Reply:
oops, this needs to be cleaned up 🤦 I agree that linter may be able to detect.